vignettes/ebola.Rmd

```diff
@@ -212,8 +212,8 @@ Now we are ready to fit the latent individual model.
 
 We start by fitting a single lognormal distribution to the data. 
 This model assumes that a single distribution describes all delays in the data, regardless of the case's location, sex, or any other covariates.
-To do this, we set `formula = bf(mu ~ 1, sigma ~ 1)` to place an model with only and intercept parameter (i.e. `~ 1` in R formula syntax) on the `mu` and `sigma` parameters of the lognormal distribution.
-This distribution is specified using `family = lognormal()`.
+To do this, we set `formula = bf(mu ~ 1, sigma ~ 1)` to place an model with only and intercept parameter (i.e. `~ 1` in R formula syntax) on the `mu` and `sigma` parameters of the lognormal distribution specified using `family = lognormal()`.
```

Comment:
I think here and in the getting started we should introduce new concepts when we need them (i.e when we put a model on sigma later here) vs at the start.

Reply:
I think that if we change the text here we should still mention that there is a model placed on `sigma`

Reply:
[0e11754](https://github.com/epinowcast/epidist/pull/396/commits/0e1175424d1f7436b3a311492b871e0c11a386ca)